docs/openapi.md

```
@@ -1105,6 +1132,634 @@ Examples
+}
+```
+## GET `/v1/prompts`
+
+> **List Prompts Handler**
+
+Handle requests to the GET /prompts endpoint.
+
+Process GET requests that list all stored prompt templates from the Llama
+Stack service. For example:
+
+    curl http://localhost:8080/v1/prompts
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)